### PR TITLE
Add forwarding to the language repository.

### PR DESCRIPTION
**Pull request by lrhn**
Starting with `language/` or just `l/` forwards the non-triage/non-label related operations to the language repository instead of the SDK repository.

### CODE REVIEW

**Comment by lrhn** [comment]
WDYT?

**Comment by kevmoo** [comment]
Sure!

File: lib/redirect.dart [hunk 1]
```diff
@@ -147,8 +147,9 @@ final _matchers = <RegExp, Uri Function(Match)>{
 /// and [base2] otherwise.
 /// (The former is an SDK repo base, the latter a language repo base.)
 Uri Function(Match) _resolveLastChoose(Uri base1, Uri base2) => (Match match) {
-      return ((match[1] == null) ? base1 : base2)
-          .resolve(match[match.groupCount]!);
+      return ((match[1] == null) ? base1 : base2).resolve(
+        match[match.groupCount]!,
```
Comment (kevmoo) [inline lib/redirect.dart:151]: yes! is so nice! 😁

**Review by kevmoo — APPROVED** [review]

**Comment by kevmoo** [comment]
@lrhn – the cool thing. It'll auto deploy as soon as you land this 😁

**Comment by lrhn** [comment]
Cool thing happening!

**Pull request merged**